packages/synapse-react-client/src/utils/functions/TableColumnSchemaUtils.test.ts

```diff
@@ -93,6 +143,42 @@ describe('TableColumnSchemaUtils', () => {
         newColumnId: NEW_COLUMN_MODEL_ID,
       })
     })
+    test('add new column that has an existing ID', async () => {
```

Comment:
Remove this test case or decide if it's possible to throw an error, because this is an invalid scenario